jsapp/js/components/common/koboDropdown.tsx

```diff
@@ -1,27 +1,52 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import autoBind from 'react-autobind';
 import bem, {makeBem} from 'js/bem';
 import {
-  createEnum,
   KEY_CODES,
 } from 'js/constants';
 import koboDropdownActions from './koboDropdownActions';
 import './koboDropdown.scss';
 
-export const KOBO_DROPDOWN_THEMES = createEnum([
-  'light',
-  'dark',
-]);
+export enum KOBO_DROPDOWN_THEMES {
+  light = 'light',
+  dark = 'dark',
+}
+
+export enum KOBO_DROPDOWN_PLACEMENTS {
+  'up-left' = 'up-left',
+  'up-center' = 'up-center',
+  'up-right' = 'up-right',
+  'down-left' = 'down-left',
+  'down-center' = 'down-center',
+  'down-right' = 'down-right',
+}
 
-export const KOBO_DROPDOWN_PLACEMENTS = createEnum([
-  'up-left',
-  'up-center',
-  'up-right',
-  'down-left',
-  'down-center',
-  'down-right',
-]);
+type KoboDropdownThemesType = 'light' | 'dark'
+
+type KoboDropdownPlacementsType =
+  'up-left' |
+  'up-center' |
+  'up-right' |
+  'down-left' |
+  'down-center' |
+  'down-right'
+
+
+type KoboDropdownProps = {
+  theme: KoboDropdownThemesType,
+  placement: KoboDropdownPlacementsType,
```

Comment:
You don't need `KoboDropdownThemesType` and `KoboDropdownPlacementsType` - you can use enum here. Enums in TypeScript are both a data to be used in code and a type. See `textBox.tsx` for identical situation :)